spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,75 @@
+<pre class='metadata'>
+Title: Web Periodic Background Synchronization
+Status: CG-DRAFT
+ED: https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.html
+Shortname: periodic-background-sync
+Level: 1
+Editor: Mugdha Lakhani, Google, nator@chromium.org
+Abstract: This specification describes a method that enables web applications to periodically synchronize data and content in the background.
+Group: wicg
+Repository: WICG/BackgroundSync
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:dfn; for:/; text:browsing context
+spec:service-workers; type:dfn; text:frame type
+</pre>
+
+Introduction {#intro}
+=====================
+
+  <em>This section is non-normative.</em>
+
+  Web Applications often run in environments with unreliable networks (e.g., mobile phones) and unknown lifetimes (the browser might be killed or the user might navigate away).
+  This makes it difficult for web apps to keep their content and state in sync with servers. This API is intended to reduce the time between content creation and content synchronization between the servers and the web app.
```

Comment:
This is still very abstract, please add one or two really clear, clarifying sentences on _what_ periodic background sync is, and how it helps solving that problem.

Reply:
done.